templates/product-analytics/next/package.json

```diff
@@ -9,16 +9,22 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@514labs/moose-lib": "^0.0.0",
```

Comment:
This is probably not going to select any version.

![Screenshot 2024-06-19 at 1 28 31 PM](https://github.com/514-labs/moose/assets/4429209/2a6a626f-998b-4ffb-952c-d5c82311987f)